src/components/forms/fragments/overview/bigShipsInfo/bigShipsInfo.scss

```diff
@@ -0,0 +1,8 @@
+@import 'styles/variables';
+
+.vene-big-ships-info {
+  &__header {
+    padding-top: $spacing-01;
+    padding-bottom: $spacing-00-50;
```

Comment:
Are those naming follow any convention ?  If yes, can you paste refer link?

Reply:
Actually, there is not, I came up with these names. But I did read the idea of having limited values for spacing a while ago in a book, and I'm following this principle since then.
For smaller values, gaps are small and they grow proportionately. like: 
0.25, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 6, 8, 12, 16, 24, 32, 40, 48.